src/tests/dictionaryConcepts/components/CreateMapping.test.js

```diff
@@ -45,4 +49,31 @@ describe('Test suite for dictionary concepts components', () => {
     inputField.find('#remove').first().simulate('click');
     expect(props.removeMappingRow).toHaveBeenCalled();
   });
+
+  it('should render when isNew is true and source not CIEL', () => {
+    const newProps = {
+      ...props,
+      isNew: true,
+      source: 'Snomed',
+    };
+
+    wrapper = mount(<Router>
+      <table><tbody><CreateMapping {...newProps} /></tbody></table>
+    </Router>);
+    const inputs = wrapper.find('input');
+    expect(inputs).toHaveLength(3);
+  });
+
+  it('should render when isNew is true and source equal to CIEL', () => {
+    const newProps = {
+      ...props,
```

Comment:
Where are you passing the source?

Reply:
`source` is defined in the `props`, at the beginning of this file